lib/jekyll/assets/liquid/tag/proxied_asset.rb

```diff
@@ -16,18 +16,20 @@ class ProxiedAsset
 
           # ------------------------------------------------------------------
 
+          attr_reader :content_type
+
           def_delegator :@asset, :liquid_tags
-          def_delegator :@asset, :filename, :source_filename
-          def_delegator :@asset, :content_type
+          def_delegator :@asset, :source_filename
           def_delegator :@asset, :mtime
 
           # ------------------------------------------------------------------
 
           def initialize(asset, args, env, tag)
             @env = env
-            @asset = asset
+            @asset = asset.dup
```

Comment:
> cdf6de7266f1da9cfa9662b9e9bd29815a7b1270 Proxies: fix mutable content type affecting later use of asset

Changing the content type once will affect it in the future. I'm just fixing up an alternative to this which isn't so expensive.